sites/wonderland/docs/development/solidity/onboarding/knowledge-base/recommended-resources.md

```diff
@@ -0,0 +1,45 @@
+---
+    title: Recommended Resources
+---
+
+Working in DeFi often means standing on the shoulders of giants and using existing code to speed up development. Some contracts are forked more often than others, and with time, they become famous or influential.
+
+:::tip Ecosystem Awareness
+Understanding these foundational contracts and standards is crucial for DeFi development. When you encounter similar patterns in protocols, you will recognize the battle-tested approaches that have shaped the ecosystem.
+:::
+
+## 🏗️ Foundational Contracts
+
+These are the contracts that have shaped DeFi and continue to influence new protocols:
+
+### Core Infrastructure
+- [Chainlink](https://docs.chain.link/data-feeds) - The global standard for blockchain connectivity.
+- [Connext](https://docs.connext.network/) - Cross-chain communication protocol.
```

Comment:
Connext changed to Everclear we should update this, maybe remove it? or move it to a new section DeFi ?